kajas/src/app/add-artwork/add-artwork.component.ts

```diff
@@ -1,10 +1,115 @@
 import { Component } from '@angular/core';
+import { FormBuilder, FormGroup, Validators } from '@angular/forms';
+import { HttpClient } from '@angular/common/http';
+import { Router } from '@angular/router';
+import { SessionStorageService } from 'angular-web-storage';
+import axios from 'axios'; 
 
 @Component({
   selector: 'app-add-artwork',
   templateUrl: './add-artwork.component.html',
-  styleUrl: './add-artwork.component.css'
+  styleUrls: ['./add-artwork.component.css']
 })
 export class AddArtworkComponent {
+  artworkForm: FormGroup;
+  imageUrl: string | ArrayBuffer | null = null;
+  selectedFile: File | null = null;
+  showModal = false;
+  modalMessage = '';
 
+  constructor(
+    private fb: FormBuilder, 
+    private http: HttpClient,
+    private router: Router, 
+    private sessionStorage: SessionStorageService
+  ) {
+    this.artworkForm = this.fb.group({
+      title: ['', Validators.required],
+      date: ['', Validators.required],
```

Comment:
Always use the object form in creating validators.
[https://docs.google.com/presentation/d/1Mdcq3WcHksBlbCvxi6wx5vajn4v8LGlJRnBIu3x4fvw/edit#slide=id.g270ee9174f4_0_43](https://github.com/PUP-BSIT/project-midknight_appdev/pull/25/url)